### PR TITLE
Missed python module definition and setup.py script

### PR DESCRIPTION
**Pull request by ipa-nhg**
Related to https://github.com/ros-industrial/universal_robot/issues/293

### CODE REVIEW

File: ur_kinematics/src/ur_kinematics/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from ur_kinematics import *
```
Comment (Chamango90) [inline ur_kinematics/src/ur_kinematics/__init__.py]: The import is not required. An empty `__init__.py` file is sufficient.

Comment (ipa-nhg) [reply ur_kinematics/src/ur_kinematics/__init__.py]: you are right, it is within the same package. The __init__ file can be empty

**Comment by gavanderhoorn** [comment]
Suggestion: can we please rebase as much as possible, instead of merging in `kinetic-devel` all the time? Let's try to keep the commit history sane.

**Comment by ipa-nhg** [comment]
@gavanderhoorn yes! sorry, I clicked the button "update branch" of the PR  to verify the commit with Travis. But you are right, it is horrible for the history of the repo...

**Comment by gavanderhoorn** [comment]
No need to apologise.

Let's just keep it in mind.

**Comment by Chamango90** [comment]
Reviewed and accepted

**Comment by gavanderhoorn** [comment]
Thanks @ipa-nhg for fixing this.

**Comment by pidipidi** [comment]
I cannot still find a python module, ur_kin_py when I run test_analytical_ik.py. Here is the message,

"ImportError: No module named ur_kin_py"

I am not an expert for the python wrapper, but I could not find any setup for ur_kin_py.cpp in CMakeList.

**Comment by cambel** [comment]
@pidipidi Yeah I found the same error, after trying many things I manage to made it work.

Here is what I `did`:
First, make sure you have installed Boost.Numpy, if not you can install it from source:
```
sudo apt-get install build-essential python-setuptools libatlas-dev libatlas3gf-base

git clone https://github.com/ndarray/Boost.NumPy
cd Boost.NumPy
cmake .
make
make install
```

Then in the CMakeLists.txt file, you have to add the following

```
find_package(PythonLibs 2.7 REQUIRED)
... 
###########
## Build ##
###########
...
PYTHON_ADD_MODULE(ur_kin_py src/ur_kin_py.cpp)
target_link_libraries(ur_kin_py ur3_kin boost_python boost_numpy ${PYTHON_LIBRARIES} ${Boost_LIBRARIES} )

set_target_properties(ur_kin_py PROPERTIES 
                      LIBRARY_OUTPUT_DIRECTORY ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_PYTHON_DESTINATION})
...
#############
## Install ##
#############
install(TARGETS ur_kin_py
  ARCHIVE DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}
  LIBRARY DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}
)
...
```
Note that I link the library just with the ur3_kin, if you need a different robot, you can change it.

Then you can test with the `test_analytical_ik.py` but you have to also fix the import line on this file:
`from ur_kinematics.ur_kin_py import forward, inverse`
then it should work fine.

Finally, if you encounter the problem of `libboost_numpy.so: cannot open shared object file` just update your LD_LIBRARY_PATH with the location of your library, in my case /usr/local/lib64

Hope it helps!

**Pull request merged**